### PR TITLE
Change oil to canols

### PR DESCRIPTION
**Pull request by JayRuiz**

### CODE REVIEW

File: cake-recipe.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ Ingredients:
  - 1 1/2 teaspoons baking soda
  - 1 teaspoon salt
  - 2 eggs
- - 1 cup milk
+ - 1 cup soil milk
```
Comment (JayRuiz) [inline cake-recipe.txt]: I JayRuiz made typo on 'soil' instead of soy

Comment (RuizJay) [inline cake-recipe.txt]: Hi JayRuiz, you made typo on "Soil' instead of soy.

File: cake-recipe.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canols oil
```
Comment (RuizJay) [inline cake-recipe.txt]: typo "canols" please fix this.

**Comment by JayRuiz** [comment]
RuizJay please check out the final status

**Comment by RuizJay** [comment]
JayRuiz it looks good.

**Pull request merged**